examples/components/RadarDemo.vue

```diff
@@ -84,21 +48,20 @@ export default {
       radius: 100,
       angle: 60,
       direction: 90,
-      draggable: true,
+      draggable: true,  // todo: after dragging, changing direction and angle makes radar to go back initial position
```

Comment:
Regarding previous comment, just verify you really need it to be draggable before you fix the bug.

Reply:
Yeah I really need it to be draggable and rotatable :)